lib/ace/ext/papeeria/papeeria_latex_highlight_rules.coffee

```diff
@@ -142,55 +210,47 @@ define((require, exports, module) ->
         ]
         regex: "(\\\\(?:begin|end))({)(\\w*)(})"
       }
+    ])
+
+    @$rules[LIST_ITEMIZE_STATE] = [].concat(equationStartRules, listStartRules, citationsRules, [
+      endRule(LIST_ITEMIZE_REGEX)
       {
         token: [
```

Comment:
This object seems to be duplicated across a few rules. Can we create a single one?